src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3604,44 +3611,68 @@ const static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> otrHandlers = {
     { G_SETINTENSITY, gfx_set_intensity_handler_custom },            // G_SETINTENSITY (0x40)
 };
 
-const static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> f3dex2Handlers = {
+#ifdef F3DEX_GBI_2
+const static std::unordered_map<int8_t, GfxOpcodeHandlerFunc> f3dex2Handlers = {
     { G_NOOP, gfx_noop_handler_f3dex2 },
     { G_CULLDL, gfx_cull_dl_handler_f3dex2 },
     { G_MARKER, gfx_marker_handler_f3dex2 },
     { G_INVALTEXCACHE, gfx_invalidate_tex_cache_handler_f3dex2 },
     { G_MTX, gfx_mtx_handler_f3dex2 },
     { G_POPMTX, gfx_pop_mtx_handler_f3dex2 },
-#ifdef F3DEX_GBI_2
     { G_MOVEMEM, gfx_movemem_handler_f3dex2 },
     { G_MOVEWORD, gfx_moveword_handler_f3dex2 },
-#else
-    { G_MOVEMEM, gfx_movemem_handler_f3d },
-    { G_MOVEWORD, gfx_moveword_handler_f3d },
-#endif
     { G_TEXTURE, gfx_texture_handler_f3dex2 },
+    { G_GEOMETRYMODE, gfx_geometry_mode_handler_f3dex2 },
+    { G_QUAD, gfx_quad_handler_f3dex2 },
+    { G_SETOTHERMODE_L, gfx_othermode_l_handler_f3dex2 },
+    { G_SETOTHERMODE_H, gfx_othermode_h_handler_f3dex2 },
     { G_VTX, gfx_vtx_handler_f3dex2 },
     { G_MODIFYVTX, gfx_modify_vtx_handler_f3dex2 },
     { G_DL, gfx_dl_handler_common },
     { G_DL_INDEX, gfx_dl_index_handler },
     { G_PUSHCD, gfx_pushcd_handler_f3dex2 },
     { G_ENDDL, gfx_end_dl_handler_common },
-#ifdef F3DEX_GBI_2
-    { G_GEOMETRYMODE, gfx_geometry_mode_handler_f3dex2 },
-#else
+    { G_TRI1, gfx_tri1_handler_f3dex2 },
+    { G_TRI2, gfx_tri2_handler_f3dex },
+    // Commands to implement
+    { G_SPNOOP, gfx_spnoop_command_handler_f3dex2 },
+    { G_RDPHALF_1, gfx_stubbed_command_handler },
+};
+#endif
+
+const static std::unordered_map<int8_t, GfxOpcodeHandlerFunc> f3dexHandlers = {
+    { G_NOOP, gfx_noop_handler_f3dex2 },
+    { G_CULLDL, gfx_cull_dl_handler_f3dex2 },
+    { G_MARKER, gfx_marker_handler_f3dex2 },
+    { G_INVALTEXCACHE, gfx_invalidate_tex_cache_handler_f3dex2 },
+    { G_MTX, gfx_mtx_handler_f3d },
+    { G_POPMTX, gfx_pop_mtx_handler_f3d },
+    { G_MOVEMEM, gfx_movemem_handler_f3d },
+    { G_MOVEWORD, gfx_moveword_handler_f3d },
+    { G_TEXTURE, gfx_texture_handler_f3d },
+    { G_SETOTHERMODE_L, gfx_othermode_l_handler_f3d },
+    { G_SETOTHERMODE_H, gfx_othermode_h_handler_f3d },
     { G_SETGEOMETRYMODE, gfx_set_geometry_mode_handler_f3dex },
     { G_CLEARGEOMETRYMODE, gfx_clear_geometry_mode_handler_f3dex },
```

Comment:
`error: ‘G_CLEARGEOMETRYMODE’ was not declared in this scope; did you mean ‘G_EXTRAGEOMETRYMODE’?`